### DISCUSSION TITLE
Firmware Retraction

### DISCUSSION THREAD

**Discussion by JoeMaxey** [mriscoc/Ender3V2S1#27] [Q&A] [answered]
I'm running a direct drive and set my retraction to 1mm in Cura. I noticed in the Tune menu during a print that I could tune the retraction. I also noticed it said my current retraction was 5mm, even though it was set to 1mm in Cura. Do you know which retraction setting the print is actually using?

**Comment by mriscoc** [comment, marked as answer]
The retraction settings in "Tune" and "Filament Settings" menus are for an advanced feature called [Firmware Retraction](https://github.com/mriscoc/Ender3V2S1/wiki/Firmware-Retraction) that allows you to tunning the best retraction for improve quality and print speed. You must to configure your slicer according to the Wiki to enable that feature.
